### PR TITLE
Fix issues with shebang

### PR DESCRIPTION
**Pull request by gmoro**
Also, fix typo on README

### CODE REVIEW

File: ova-compose/ova-compose.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
```
Comment (oliverkurth) [inline ova-compose/ova-compose.py:1]: No. This prevents users from using another python installation.

What's the reason for this change?

Comment (gmoro) [reply ova-compose/ova-compose.py:1]: sorry missed the original comment

Change shebang as it causes trouble when packaging the software as RPM.

Example of RPM linting:
[ 4s] open-vmdk.x86_64: E: env-script-interpreter (Badness: 9) /usr/bin/ova-compose /usr/bin/env python3
[ 4s] This script uses 'env' as an interpreter. For the rpm runtime dependency
[ 4s] detection to work, the shebang #!/usr/bin/env needs to be
[ 4s] patched into #!/usr/bin/ otherwise the package dependency
[ 4s] generator merely adds a dependency on /usr/bin/env rather than the actual
[ 4s] interpreter /usr/bin/. Alternatively, if the file should not be
[ 4s] executed, then ensure that it is not marked as executable or don't install it
[ 4s] in a path that is reserved for executables.

I'm patching it for now in my package, but just now I'm thinking this might break certain installs, will dig further to see if there's any other way to avoid the rpmlint error maintaining a more generic approach

Comment (oliverkurth) [reply ova-compose/ova-compose.py:1]: That's a weird error. `/usr/bin/` is not an interpreter, it's a directory. And using `#!/usr/bin/env python3` is a very common pattern in python scripts. What application are you using here?

If you need an RPM spec file, you can use this for starters:
https://github.com/vmware/photon/blob/master/SPECS/open-vmdk/open-vmdk.spec

Comment (oliverkurth) [reply ova-compose/ova-compose.py:1]: Okay, you should use backticks when quoting text - the original text from `rpmlint` says `#!/usr/bin/<interpreter> `. The `<interpreter>` part isn't visible, so I thought it's a directory.

```
This script uses 'env' as an interpreter. For the rpm runtime dependency
detection to work, the shebang #!/usr/bin/env <interpreter>  needs to be
patched into #!/usr/bin/<interpreter>  otherwise the package dependency
generator merely adds a dependency on /usr/bin/env rather than the actual
interpreter /usr/bin/<interpreter>.  Alternatively, if the file should not be
executed, then ensure that it is not marked as executable or don't install it
in a path that is reserved for executables.
```

Comment (gmoro) [reply ova-compose/ova-compose.py:1]: Yeah, sorry, didn't notice.
I will update my package with your spec, that's really appreciated.
By the way, that's where I'm building it
https://build.opensuse.org/package/show/home:gmoro:openvmdk/open-vmdk

Comment (oliverkurth) [reply ova-compose/ova-compose.py:1]: I missed your last comment, and the link isn't working any more - but I found it here: https://software.opensuse.org/package/open-vmdk

Thanks for packaging! One question - why is the version `0.3.1.0-1.1` and not just `0.3.1-1.1` (why the extra `0`)?

Comment (oliverkurth) [reply ova-compose/ova-compose.py:1]: I fixed the typo in the README, thank you!

I am going to close the PR. Fixing the shebang probably makes sense for packaging since you know where python is installed, but generally `#!/usr/bin/env python3` is correct.
